src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoService.scala

```diff
@@ -61,6 +66,19 @@ class LeonardoService(protected val dataprocConfig: DataprocConfig, gdDAO: Datap
     }
   }
 
+
+  def processClusterRequest(googleProject: GoogleProject, clusterName: ClusterName, clusterRequest: ClusterRequest): ClusterRequest = {
+    // create a LabelMap of default labels
+    val defaultLabels = DefaultLabels(clusterName, googleProject, clusterRequest.bucketPath, clusterRequest.serviceAccount, clusterRequest.jupyterExtensionUri)
+      .toJson.asJsObject.fields.mapValues(labelValue => labelValue.toString.replaceAll("\"", ""))
```

Comment:
Why `.replaceAll("\"", "")`? Are quotes being injected somehow?

Reply:
There seemed to be extra quotes around the values (not the keys though) - but I experimented a bit more and using `labelValue.convertTo[String]` actually gets rid of the extra quotes, so I'll use that